.idea/cookbook.iml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like you have a bunch of `.idea` folders in here. Probably remove them and then add `.idea/**/*` to .gitignore.

Also rebase out your commits into 1.

Reply:
Thanks for your patience :)